tests/rptest/clients/offline_log_viewer.py

```diff
@@ -37,6 +37,16 @@ def _json_cmd(self, node, suffix):
             self._redpanda.logger.error(f"Invalid JSON output: {json_out}")
             raise
 
+    def inspect_data_directory(self, node):
+        cmd = f"python3 /opt/scripts/offline_log_viewer/data_directory_inspector.py --path {self._redpanda.DATA_DIR}"
```

Comment:
why not invoke this as a module rather than shell output. i.e. `import offline_log_viewer.data_directory.main()`

Reply:
it is being executed on the remote node: `node.account.ssh_output(cmd, combine_stderr=False)`